### PR TITLE
Update release notes for 0.18.0

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -11,7 +11,26 @@ Array
 - Fix ``rechunk`` with chunksize of -1 in a dict (:pr:`3469`) `Stephan Hoyer`_
 - Add to/read_zarr for Zarr-format datasets and arrays (:pr:`3460`) `Martin Durant`_
 - Experimental addition of generalized ufunc support, ``apply_gufunc``, ``gufunc``, and
-  ``as_gufunc`` (:pr:`#3109`) `Markus Gonser`_
+  ``as_gufunc`` (:pr:`3109`) (:pr:`3526`) (:pr:`3539`) `Markus Gonser`_
+- Faster slice_1d in dask.array (:pr:`3479`) `Yu Feng`_
+- Add split_every parameter to einsum (:pr:`3472`) `Guido Imperiale`_
+- Avoid unnecessary rechunking tasks (:pr:`3529`) `Matthew Rocklin`_
+- add to/read_zarr (:pr:`3460`) `Matthew Rocklin`_
```
Comment (jakirkham) [inline docs/source/changelog.rst]: Should be Martin's, right? ;)

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -11,7 +11,25 @@ Array
 - Fix ``rechunk`` with chunksize of -1 in a dict (:pr:`3469`) `Stephan Hoyer`_
 - Add to/read_zarr for Zarr-format datasets and arrays (:pr:`3460`) `Martin Durant`_
 - Experimental addition of generalized ufunc support, ``apply_gufunc``, ``gufunc``, and
-  ``as_gufunc`` (:pr:`#3109`) `Markus Gonser`_
+  ``as_gufunc`` (:pr:`3109`) (:pr:`3526`) (:pr:`3539`) `Markus Gonser`_
+- Faster slice_1d in dask.array (:pr:`3479`) `Yu Feng`_
+- Add split_every parameter to einsum (:pr:`3472`) `Guido Imperiale`_
+- Avoid unnecessary rechunking tasks (:pr:`3529`) `Matthew Rocklin`_
+- Compute dtypes at runtime for fft (:pr:`3511`) `Matthew Rocklin`_
+- Include "region" in key names for da.store (:pr:`3540`) `Martin Durant`_
```
Comment (jakirkham) [inline docs/source/changelog.rst]: This probably should have a better name (though I understand it is coming from the PR title). It should be something like, "Generate UUIDs for all `da.store` operations" or similar.

Comment (jakirkham) [reply docs/source/changelog.rst]: Thoughts @martindurant?

**Comment by jakirkham** [comment]
One question above, otherwise LGTM.

**Comment by mrocklin** [comment]
Can I ask you to edit directly? I'm unlikely to expend much effort fine
tuning the release notes line by line.

**Comment by mrocklin** [comment]
Merged manually.  Closing

**Pull request closed without merging**